packages/sdk/src/FlagResolverClient.ts

```diff
@@ -103,13 +104,18 @@ export class FetchingFlagResolverClient implements FlagResolverClient {
     // todo refactor to move out environment
     environment,
     region,
+    customResolveBaseUrl,
   }: FlagResolverClientOptions) {
     // TODO think about both resolve and apply request logic for backends
     this.fetchImplementation = environment === 'backend' ? fetchImplementation : withRequestLogic(fetchImplementation);
     this.clientSecret = clientSecret;
     this.sdk = sdk;
     this.applyTimeout = applyTimeout;
-    this.baseUrl = region ? `https://resolver.${region}.confidence.dev/v1` : 'https://resolver.confidence.dev/v1';
+    if (environment === 'backend' && customResolveBaseUrl) {
+      this.baseUrl = customResolveBaseUrl.concat('/v1');
```

Comment:
Use a template string instead. The concat method is never used in JS. Concat is usually done with the plus operator, but if you do that the linter will instead turn it into a template string which is the de facto standard for all string operations now days.
```suggestion
      this.baseUrl = `${customResolveBaseUrl}/v1`;
```

Reply:
Oh so this is how it's better done 😮